src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+
+export const Person = ({ person}) => {
+  
+  const { name, age, sex, isMarried, partnerName } = person;
+  return (
```

Comment:
```suggestion
export const Person = ({ person}) => {
  const { name, age, sex, isMarried, partnerName } = person;
  
  return (
```
Add a blank line between only between statements